### PR TITLE
Switch styled

### PR DESCRIPTION
**Pull request by thomasgwatson**
Merging straight into dev, since these changes only impact Storybook.

Just wanted to knock this component out in some spare time. The original switch has a different look and feel and doesn't have color customization; as it is already in use, I decided to build out a new switch and leave the old in place.

Based on https://www.figma.com/file/lh2DVGMh7CRdUsxF3k6vGI/Map-Improvements?node-id=0%3A1

Screen Shot 2020-05-25 at 14 23 04
Screen Shot 2020-05-25 at 14 22 59

### CODE REVIEW

File: src/components/Switch/Switch.stories.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    setSwitchBool(!switchBool)
+  }
+  return (<Switch value={switchBool} onClick={handleToggleSwitch} />)
+}
+
+function SwitchStyledDemo () {
+  const [switchBool, setSwitchBool] = useState(false)
+
+  function handleToggleSwitch (checked, name) {
+    console.log(checked, name)
```
Comment (tibetsprague) [inline src/components/Switch/Switch.stories.js:27]: remove console

File: src/components/SwitchStyled/SwitchStyled.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <span styleName={cx('button', { buttonChecked: checked })} />
+    </div>
+  )
+}
+
+SwitchStyled.defaultProps = {
+  checked: false,
+  disabled: false,
+  backgroundColor: '#ff44ff',
+  onChange: (checked, name) => { console.log(checked, name) }
```
Comment (tibetsprague) [inline src/components/SwitchStyled/SwitchStyled.js:23]: remove console

Comment (thomasgwatson) [reply src/components/SwitchStyled/SwitchStyled.js:23]: This is a default function, to help developers as they are setting up their toggles. The alternative is a noop: () => {}. Would you prefer that?

File: src/components/Switch/Switch.stories.js (hunk 1 above)
Comment (thomasgwatson) [reply src/components/Switch/Switch.stories.js:27]: This is in the storybook, it is logging just to demonstrate how the component works

Comment (tibetsprague) [reply src/components/Switch/Switch.stories.js:27]: ok, fair enough

File: src/components/SwitchStyled/SwitchStyled.js (hunk 2 above)
Comment (tibetsprague) [reply src/components/SwitchStyled/SwitchStyled.js:23]: hmm, not sure, I guess you're right, this could be useful. go ahead with it

**Pull request merged**